config/job.js

```diff
@@ -100,6 +100,5 @@ module.exports = {
     image: SCHEMA_IMAGE,
     job: SCHEMA_JOB,
     settings: SCHEMA_SETTINGS,
-    template: SCHEMA_TEMPLATE,
-    fullstep: SCHEMA_FULL_STEP_OBJECT
```

Comment:
This removes it from the `module.exports`, but it [persists here](https://github.com/screwdriver-cd/data-schema/blob/master/config/job.js#L60). Do we still want to include this in the [`SCHEMA_STEP`](https://github.com/screwdriver-cd/data-schema/blob/master/config/job.js#L68)?